Updates and corrections to Adobe-generated alert policy info (#6078)
magento/devdocs: guides/v2.2/cloud/project/new-relic.md

## Monitor performance with alert policies

> Adobe has generated a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:

Author: ```suggestion
Adobe provides a generated a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:
```

> Adobe has generated a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:

Author: ```suggestion
Adobe provides a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:
```

> Adobe provides a generated a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:

Author: ```suggestion
Adobe provides a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:
```

{:.procedure}
To review Adobe-generated alert policies:

1. Use your Project Owner New Relic credentials to [log in to your New Relic account](https://login.newrelic.com/login).

> 1. From the navigation  menu, select  **Alerts**  >  **Alert Policies**.

Reviewer: ```suggestion
1. From the navigation menu, select  **Alerts**  > **Alert Policies**.
```
